app/components/pages/createPoll/styles.js

```diff
@@ -0,0 +1,16 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  CustomFullWidth: {
+    width: '100%',
+  },
+  buttonSize: {
+    width: '140Px',
+    height: '40px',
+  },
+  errorColor: {
```

Comment:
you're not using it so remove it 
also if you ever want to change any element color use color form theme

Reply:
okay